src/fetchers/simplehash.js

```diff
@@ -4,28 +4,24 @@ import { Interface } from "ethers/lib/utils";
 import slugify from "slugify";
 
 import { parse } from "../parsers/simplehash";
-import { getProvider } from "../shared/utils";
+import { chains, getProvider } from "../shared/utils";
 import { logger } from "../shared/logger";
 import _ from "lodash";
 
 const getNetworkName = (chainId) => {
-  let network;
-  if (chainId === 1) {
-    network = "ethereum";
-  } else if (chainId === 10) {
-    network = "optimism";
-  } else if (chainId === 56) {
-    network = "bsc";
-  } else if (chainId === 137) {
-    network = "polygon";
-  } else if (chainId === 42161) {
-    network = "arbitrum";
-  } else if (chainId === 5) {
-    network = "ethereum-goerli";
-  } else {
+  const network = Object.keys(chains).find(key => chains[key] === chainId);
```

Comment:
Does it make sense for it to be a dictionary? that way you can just do chains[key]?

Reply:
it is a dictionary but in collection.js/token.js we need the network name as the key, and here it's the other way around. could export two dictionaries i guess?

Reply:
i see, no need